app/models/grape.rb

```diff
@@ -3,11 +3,12 @@ def cut
     "cut cut!"
   end
 
-  def slice
+  def slice(params)
```

Comment:
__[method_definition_validator]__
* Lint/DefinitionValidator: Not enough arguments. Did you forget the following arguments? [params] at [app/models/grape.rb#L11](/actcat/sideci_rails_sandbox/blob/8f814ddecda9c71f1321383f85407d15ea95f368/app/models/grape.rb#L11)